src/Utils/Image.php

```diff
@@ -321,15 +321,15 @@ public function resize($width, $height, $flags = self::FIT)
 	 */
 	public static function calculateSize($srcWidth, $srcHeight, $newWidth, $newHeight, $flags = self::FIT)
 	{
-		if (substr($newWidth, -1) === '%') {
-			$newWidth = round($srcWidth / 100 * abs($newWidth));
+		if (substr((string) $newWidth, -1) === '%') {
+			$newWidth = (int) round($srcWidth / 100 * abs($newWidth));
 			$percents = TRUE;
 		} else {
 			$newWidth = (int) abs($newWidth);
 		}
 
-		if (substr($newHeight, -1) === '%') {
-			$newHeight = round($srcHeight / 100 * abs($newHeight));
+		if (substr((string) $newHeight, -1) === '%') {
+			$newHeight = (int) round($srcHeight / 100 * abs($newHeight));
```

Comment:
Casting `round` to int? Why not just cast the expression, i.e. `(int) ($srcHeight / 100 * abs($newHeight))`?